htdocs/lostpass.php

```diff
@@ -97,7 +97,7 @@
 		$xoopsMailer->setToUsers($getuser[0]);
 		$xoopsMailer->setFromEmail($icmsConfig['adminmail']);
 		$xoopsMailer->setFromName($icmsConfig['sitename']);
-		$xoopsMailer->setSubject(sprintf(_US_NEWPWDREQ, ICMS_URL));
+		$xoopsMailer->setSubject(sprintf(_US_NEWPWDREQ, $icmsConfig['sitename'] . ' (' . ICMS_URL . ')'));
```

Comment:
I think it would be more correctly to add ICMS_URL into language file as new variable than doing that here. 

That constant comes from user.php -> https://github.com/ImpressCMS/impresscms/blob/eb9403215854d733d01a2f73c5370d81cd9ca2dc/htdocs/language/english/user.php#L89 

So change that line into: 
```php
define('_US_NEWPWDREQ','New Password Request at %s (%s)');
```

And here:
```php
$xoopsMailer->setSubject(sprintf(_US_NEWPWDREQ, $icmsConfig['sitename'], ICMS_URL));
```

This would make sure that everywhere same type of text is used/displayed in same way.

Reply:
I propose this -
`define(_US_NEWPWREQ', New Password Request at . $icmsConfig['sitename'] . ' (' . ICMS_URL . ')'));`

With this, the sprintf() in lostpass.php can be eliminated.